client/src/common.rs

```diff
@@ -246,3 +249,47 @@ pub mod public_key {
         Ok(value.to_string())
     }
 }
+
+/// Handles providing the arg for and retrieval of the public key.
+pub mod public_key {
+
+    use super::*;
+
+    const ARG_NAME: &str = "public-key";
+    const IS_REQUIRED: bool = true;
+    const ARG_HELP: &str =
+        "This must be a properly formatted public key. The public key may instead be read in from \
+        a file, in which case enter the path to the file as the --public-key argument. The file \
+        should be one of the two public key files generated via the `keygen` subcommand; \
+        \"public_key_hex\" or \"public_key.pem\"";
+
+    pub(crate) fn arg(order: usize) -> Arg<'static, 'static> {
+        sealed_public_key::arg(order, ARG_NAME, ARG_HELP, IS_REQUIRED)
+    }
+
+    pub(crate) fn get(matches: &ArgMatches) -> Result<String, Error> {
+        sealed_public_key::get(matches, ARG_NAME)
+    }
+}
+
+/// Handles providing the arg for and retrieval of the session account arg when specifying an
+/// account for a Deploy.
+pub(super) mod session_account {
+    use super::*;
+
+    pub const ARG_NAME: &str = "session-account";
+    const IS_REQUIRED: bool = false;
+    const ARG_HELP: &str =
+        "This must be a properly formatted public key. The public key may instead be read in from \
```

Comment:
I think we need all this in the help string, but it should also probably start with a brief description similar to that suggested in my comment for `DeployStrParams::session_account`.

Reply:
Added the description